packages/builder/test/fixture-projects/project-with-scripts/scripts/successful-script.js

```diff
@@ -1,9 +1,11 @@
 
 const fs = require("fs");
 
-export default async function (runtimeEnv, deployer, accounts) {
+async function run(runtimeEnv, accounts, deployer) {
   if (runtimeEnv.config === undefined || runtimeEnv.network === undefined) {
     throw new Error("Config was not provided");
   }
   fs.appendFileSync("output.txt", runtimeEnv.network.name);
 }
+
+module.exports = { default: run }
```

Comment:
Why using `module.exports` rather than `export default`?

Reply:
It doesn't work the other way. This is plain JS, without any syntax additions.